tilelang/tools/lower_trace/html.py

```
@@ -0,0 +1,1728 @@
+                dot_cls = "changed"
+                status_attr = "changed"
+            else:
+                dot_cls = "noop"
+                status_attr = "noop"
+            sid = f"sec-{rec.phase}-{rec.index}"
+            stats_html = ""
+            if rec.changed:
+                stats_html = (
+                    f'<span class="pass-stats">'
+                    f'<span class="st-add">+{rec.add_lines}</span> '
+                    f'<span class="st-del">−{rec.del_lines}</span>'
```

Ruff (0.15.18)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)